Tests/Editor/PackagesTests.cs

```diff
@@ -47,5 +70,31 @@ public void InstalledPackage_IsReported(string description, string name, string
                 }
             }
         }
+
+        [Test]
+        public void RecommandedUpgradePackage_IsReproted()
+        {
+            var issuePackages = Analyze(IssueCategory.PackageVersion);
+            var matchIssue = issuePackages.FirstOrDefault(issue => issue.customProperties[0] == "com.unity.2d.pixel-perfect");
+
+            Assert.IsNotNull(matchIssue, "Cannot find the upgrade pacakge: com.unity.2d.pixel-perfect");
+            Assert.AreEqual(matchIssue.customProperties[0], "com.unity.2d.pixel-perfect");
+            Assert.AreEqual(matchIssue.customProperties[1], "3.0.2");
+            Assert.AreEqual(matchIssue.customProperties[2], "4.0.1");
```

Comment:
The recommended version will vary depending on the version of Unity. Perhaps we should just check that the recommended version is higher than 3.0.2 ?

Reply:
Hi Marco, I do not get your point. 3.0.2 is the current version. 4.0.1 is the recommended version.